app/javascript/selectors/searchUserListSelector.js

```diff
@@ -54,3 +54,7 @@ export const displayChangeLog = state => {
   }
   return true
 }
+
+export const checkSearchUserListTiles = userListTiles => {
+  return userListTiles && userListTiles.length !== 0 ? userListTiles : []
```

Comment:
Wouldn't this be the same as:

```suggestion
  return Array.isArray(userListTiles) ? userListTiles : []
```